packages/components/psammead-episode-list/src/helpers.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+// Used to make props passed to <EpisodeList> available to children
```

Comment:
which props need passed to all children? Is it `darkMode`? Wondering if there should be a Psammead Theme Context that all Psammead component can use and Simorgh can import the context provider.

Reply:
`darkMode`, `service`, `script` and `dir`

I think there should be a psammead context that passes these down - especially `service`, `script` and `dir` rather than passing it into every psammead component - not sure if it's in scope for this though - already a pretty big PR

Reply:
yeh sorry was just an idea and definitely not in scope for this PR